text/0000-compiler-team-trusted-contrib-maintainer.md

```diff
@@ -0,0 +1,401 @@
+      responsibility existing rather than being something all maintainers do.
+      Any reviewer removed can ask to be re-added when they have the bandwidth
+      to participate in FCP reviews.
+
+- Performance triage
+    - There is a rotation of maintainers and other project members who check
+      all of the interesting performance benchmarks from the last week to produce a
+      report summarizing the improvements and regressions. This is valuable to keep
+      track of the compiler's performance over time and make sure that regressions are
+      being addressed.
+
+- Issue prioritisation
+    - The compiler team has a prioritisation procedure and policy to identify
+      and label issues according to their importance. These labels feed into the
+      backport procedure (what's worth being backported) and work priorities of
+      maintainers.
+
+- Backport reviews
+    - On a regular basis, some maintainers participate in a review of pull
+      requests which have been nominated for backporting to the beta or stable
+      release. This involves a judgement call on the risk of backporting a
+      particular fix versus the severity of the issue being addressed.
+
+      Once those maintainers interested in backport reviews are identified,
+      this function could be performed in a separate meeting or asynchronously,
+      allowing the triage meeting to be streamlined and focused on nominated issues or
+      other tasks requiring broader discussion.
+
+      To establish a reasonable quorum of triage members, it is recommended that
+      at least 4 members participate in triage meetings. In the event there are not
+      enough triage members, the compiler team co-leads will act as triage members
+      until additional members are found.
+
+- Review rotation
+    - Every week, lots of pull requests are submitted to the compiler which need
+      to be reviewed. Being on the review rotation is one of the primary ways that
+      maintainers can help keep the wheels turning in the compiler team.
+
+      It is **strongly** encouraged that all maintainers be a part of the
+      review rotation.
+
+- Operations
+    - There are various *operations* tasks like agenda preparation and taking
+      meeting notes which are very useful for the team.
+
+      This list isn't exhaustive, and this RFC shouldn't be considered the canonical
+      list of these responsibilities. Similarly, this RFC isn't intended to define how
+      these responsibilities are conducted (in meetings or asynchronously, etc), that
+      should be decided and documented by those involved in each.
+
+While this RFC doesn't aim to be authoritative with respect to how maintainers
+who take on additional responsibilities are recognised, one way would be for
+maintainers who take on additional responsibilities to record this in the
+*rust-lang/team* metadata (using the `roles` key), e.g.
+
+```toml
+[people]
+members = [
```

Comment:
Responding to my above comment: having all members,l (maintainers or not) be in the compiler toml will make tracking roles easier too.

Reply:
I agree. I've removed the teams repo stuff from the RFC because I don't want to prescribe how we represent this all in that here.